sites/footers/no-footer-contextual-en.html

```diff
@@ -0,0 +1,76 @@
+---
+{
+	"title": "Hide 'Contextual' section in footer",
+	"language": "en",
+	"breadcrumbs": [
+		{ "title": "Footer", "link": "sites/footers/footers-en.html" }
+	],
+	"altLangPage": "no-footer-contextual-fr.html",
+	"secondlevel": false,
+	"dateModified": "2022-06-20",
+	"share": "true",
+	"noFooterContextual": "true"
```

Comment:
Is the `noFooterContextual` needed at all since the default bahaviour should be: 

> a page doesn't have a contextual footer unless specified

meaning that relying on the `contextualFooter` should be enough (seeing if it is set or not).

Reply:
Discuté à l'interne et nous conserverons la logique actuelle.